shed/event_streams.py

```diff
@@ -1006,22 +1006,26 @@ class Eventify(EventStream):
     >>> assert len(L) == 4
     """
 
-    def __init__(self, child, start_key, *, output_info, **kwargs):
+    def __init__(self, child, *start_keys, output_info, **kwargs):
         # TODO: maybe allow start_key to be a list of relevent keys?
-        self.start_key = start_key
-        self.val = None
+        self.start_keys = start_keys
+        self.vals = list()
         self.emit_event = False
 
         EventStream.__init__(self, child, output_info=output_info, **kwargs)
 
     def start(self, docs):
-        self.val = docs[0][self.start_key]
+        for start_key in self.start_keys:
+            self.vals.append(docs[0][start_key])
+
+        if len(self.output_info) == 1:
```

Comment:
no it doesn't. when it's length one, it assumes the output of the function should be returned as is.
this allows a function returning a tuple to have the full tuple returned.